safe_control_gym/controllers/mpc/mpc_utils.py

```diff
@@ -90,3 +92,59 @@ def reset_constraints(constraints):
     if len(constraints_list.input_state_constraints) > 0:
         raise NotImplementedError('[Error] Cannot handle combined state input constraints yet.')
     return constraints_list, state_constraints_sym, input_constraints_sym
+
+
+def set_acados_constraint_bound(constraint,
+                                bound_type,
+                                bound_value=None,
+                                ):
+    '''Set the acados constraint bound.
+
+    Note:
+        all constraints in safe-control-gym are defined as g(x, u) <= constraint_tol
+        However, acados requires the constraints to be defined as lb <= g(x, u) <= ub
+        Thus, a large negative number (-1e8) is used as the lower bound.
+        See: https://github.com/acados/acados/issues/650
+    '''
+    if bound_value is None:
+        if bound_type == 'lb':
+            bound_value = -1e8
+        elif bound_type == 'ub':
+            bound_value = 1e-6
+
+    return bound_value * np.ones(constraint.shape)
+
+
+def plot_open_loop_sol(ctrl):
+    ''' Plot the open loop predction of the MPC controller.
```

Comment:
prediction is misspelt

Reply:
Thanks for pointing this out. This will be fixed.